eng/Version-Increment.ps1

```diff
@@ -0,0 +1,90 @@
+<#
+.SYNOPSIS
+Bumps up package versions after release
+
+.DESCRIPTION
+This script bumps up package versions following conventions defined at https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#incrementing-after-release-net
+
+.PARAMETER RepoRoot
+The Root of the repo
+
+.PARAMETER ServiceDirectory
+The Name of the Service Directory
+
+.PARAMETER PackageName
+The Name of the Package
+
+.EXAMPLE
+Updating package version for core
+
+Version-Increment.ps1 -PackageCsprojPath "C:\Git\azure-sdk-for-net\sdk\core\Azure.Core\src\Azure.Core.csproj"
+
+#>
+
+[CmdletBinding()]
+Param (
+    [Parameter(Position=0)]
+    [ValidateNotNullOrEmpty()]
+    [string] $RepoRoot,
+    [string] $ServiceDirectory,
+    [string] $PackageName
+)
+# Regular expression as specified in https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+$VERSION_REGEX = "^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$"
+
+$CsprojData = New-Object -TypeName XML
+$PackageCsprojPath = Join-Path $RepoRoot "sdk" $ServiceDirectory $PackageName "src" "${PackageName}.csproj"
+$CsprojData.Load($PackageCsprojPath)
+
+$PackageVersion = Select-XML -Xml $CsprojData -XPath '/Project/PropertyGroup/Version'
+if ([System.String]::IsNullOrEmpty($PackageVersion))
+{
+    Write-Error "Package Version is missing"
```

Comment:
To help with clarity for people hitting this issue I'd suggest a message similar to "Missing Version property in project file $PackageCsprojPath".